weave/trace/serialization/op_type.py

```diff
@@ -412,14 +412,18 @@ def _get_code_deps(
                     import_code.append(import_line)
 
         else:
+            # _name will work for typing_extensions.NotRequired in 3.9
```

Comment:
This PR uses `TypedDict` and `NotRequired` as part of the markdown serializer. As part of our whole approach to custom objects, the "deserializer" Op is serialized as a backup. Serialization of the deserializer Op was failing on Python 3.9 because `NotRequired` on 3.9 requires using typing_extensions and the implementation doesn't have a `__name__` attribute.

Reply:
Nice catch!  I would put this comment in the code